mirror/cache.go

```diff
@@ -38,25 +39,32 @@ func (c *Cache) isExpired() bool {
 
 func (c *Cache) refresh() error {
 	var fs []File
-	s, err := session.NewSession(&aws.Config{
-		Region:           aws.String(c.settings.region),
-		Endpoint:         aws.String(c.settings.endpointURL),
-		S3ForcePathStyle: aws.Bool(true),
-		Credentials: credentials.NewStaticCredentials(
+	cfg, err := config.LoadDefaultConfig(context.TODO(),
+		config.WithRegion(c.settings.region),
+		config.WithEndpointResolverWithOptions(aws.EndpointResolverWithOptionsFunc(
+			func(service, region string, options ...interface{}) (aws.Endpoint, error) {
```

Comment:
This line has two issues related to code style

* `interface{}` is not used in modern Go
* the name of the variables are longer than needed

Can you refactored following this guidelines?